packages/ActionBar/README.md

```diff
@@ -40,19 +40,19 @@ npm install @paprika/action-bar
-| children         | node                                                   | false    | null                       |             |
-| columns          | arrayOf                                                | true     | -                          |             |
-| onAddFilter      | func                                                   | true     | -                          |             |
-| onApply          | func                                                   | true     | -                          |             |
-| onCancel         | func                                                   | false    | () => {}                   |             |
-| onChangeOperator | func                                                   | false    | null                       |             |
-| onClose          | func                                                   | false    | () => {}                   |             |
-| onOpen           | func                                                   | false    | () => {}                   |             |
-| operator         | [logicalFilterOperators.AND,logicalFilterOperators.OR] | false    | logicalFilterOperators.AND |             |
-| rulesByType      | objectOf                                               | false    | defaultRulesByType         |             |
+| Prop             | Type                                                 | required | default                         | Description |
+| ---------------- | ---------------------------------------------------- | -------- | ------------------------------- | ----------- |
+| appliedNumber    | number                                               | false    | 0                               |             |
+| children         | node                                                 | false    | null                            |             |
+| columns          | arrayOf                                              | true     | -                               |             |
+| onAddFilter      | func                                                 | true     | -                               |             |
+| onApply          | func                                                 | true     | -                               |             |
+| onCancel         | func                                                 | false    | () => {}                        |             |
+| onChangeOperator | func                                                 | false    | null                            |             |
+| onClose          | func                                                 | false    | () => {}                        |             |
+| onOpen           | func                                                 | false    | () => {}                        |             |
+| operator         | [Filter.types.operator.AND,Filter.types.operator.OR] | false    | Filter.types.AND                |             |
```

Comment:
why do these defaults are under Filter.types.AND instead of Filter.types.operator.AND its this expected?

Reply:
I think I must have missed it, thank you!